### PR TITLE
reviewing my work

### PR DESCRIPTION
**Pull request by misikira**

### CODE REVIEW

**Review by misikira — COMMENTED** [review]
1. good use semantic html
2. design looks good
3.  <div class="header">
             <h1>COMPUTERS AND ALL ACCESORIES</h1>
             <textarea name="enter email" rows="1" cols="40"></textarea>
             <button type="button" class="btn btn-primary">Get Started</button>

          </div>
here try to use section HTML semantic instead of using div it makes your code readable by assistive technology

**Review by misikira — COMMENTED** [review]
in my opinion, the readme file is empty try to put some info about your project to the readers this makes them understand your project better. here you can find a good template and customize it according to your  project [https://github.com/microverseinc/readme-template/blob/master/README.md](URL)

**Comment by joskalenda** [comment]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @misikira , |
|Good job so far! You still need to work on some issues to go to the next project, but you are almost there! |
|  |

## Required Changes ♻️

- [ ] Hey @misikira, thanks for making this great effort, I think you have made an awesome job, but I am sure we can do better.
- The first comment is point to a `div` tag that be replaced by a more semantic tag, the question am asking is, how should the student know this is the div you are talking about if he/she has more than one in the HTML file?... the solution to this question is using github inline comment feature, take you time to read a bit about it [here](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) and make sure you use it in the next request.
- We are not making orders when giving reviews, please make sure you give suggestions instead, `I think, in my opinion, etc...` this way the student will not feel forced to execute a certain change.
- the last one is, explaining the reason for your required changes, see you said, `the student should add some info in the readme`, okay, but why? you can explain this by sending an article's microverse link to the student on how to set up the repo in a professional way and blbalaallalaaa.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>product landing page</title>
-    <link rel="stylesheet" href="./style.css">
```
Comment (misikira) [inline index.html:6]: can you remove this stylesheet link it's duplcated

**Comment by misikira** [comment]
@joskalenda i have an issues in index.html i can't comment starting from line 10 all the way down can you help with this?

**Comment by misikira** [comment]
i put the inline commets and included the link to the reademe file template

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @misikira, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] As requested by the previous reviewer, Kindly use the following comment raised by him.

  - Make sure your comment is inform of suggestion instead of given an order.
    > N.B: 1. The student phrased the comments politely, as suggestions instead of orders. For example: This is a polite suggestion: *"I think it would be better to use semantic tags"
This reads as an order: *"Use semantic tags"
   - Also, I would suggest you give reason `WHY` you are requesting those changes.
     > N.B: The student always includes an explanation, a "why", for each requested change. For example: This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
This request lacks a "why": *Why don't you use an article tag here instead of a div?

  - I noticed you raised an issue that you could not use inline comment on your HTML file on line 10. But from my end I can see somewhere in your CSS file where you can make an inline comment. You can comment where you commented out some codes. Use this [Link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate)to learn more on how to make an inline comment


1. 
- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by misikira — COMMENTED** [review]
Hi @misikira
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in!

**Review by IngDavidTM — APPROVED** [review]
Hi @misikira  🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- It has two inline comments ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by misikira** [comment]
thank you

**Pull request merged**